apps/ecs/translations/src/en/pages.json

```diff
@@ -21,6 +21,11 @@
   },
   "confirm": {
     "header": "Summary",
+    "fields": {
+      "worker-applied-eu-settlement-scheme": {
+        "label": "Has the worker applied to the EU Settlement Scheme (EUSS)?"
```

Comment:
please add label in fields.json to render the <title> with H1.   and the same will be referred for the summary page as well. so this can be removed.